### ISSUE TITLE
top text

### ISSUE THREAD

**Issue by samlachance** [devanooga/devanooga-meta#68]
I'd like to start a discussion RE these new accounts that are joining the community for the sole purpose of entering the US job market. I have no issue with this in principle, however, I believe that qualifying this type of user as someone with an "opportunity" is kind of a stretch. It would be one thing if these were individuals who are interest in joining the community, however I believe that, in this case, devanooga is being used as an entrypoint to the US job market. Just not an appropriate use of our community IMO.

Thoughts?

**Comment by StrangeWill** [comment]
Yeah, we've been getting a lot of spam from China regarding people trying to "rent" Upwork accounts (either a scam or undermining the concept of Upwork reputation), snooping on some of the recently signed-up accounts have resulted in me ending up on sites _entirely_ dedicated to the practice (a bit of a shock to me how big of a thing this is really).

Been pondering how I want to enforce it beyond just "they're acting pretty sus" with new accounts (as is probably the one that triggered this ticket). Maybe rewrite https://www.devanooga.com/code-of-conduct/#conduct-advertising to be also encompass opportunities that don't meet a certain sniff-test? Not sure how I'd want to write that though.

**Comment by tmbeihl** [comment]
Maybe one way to deal with it is to make a rule against asking for assistance to bypass terms of service of websites or international restrictions regarding commerce?

**Comment by StrangeWill** [comment]
@tmbeihl arguably outright scams (either on the Upwork account owner, or unsuspecting customers that think they're hiring you but are actually hiring someone else), would be against our CoC by default. I'm thinking of a way to put a clamp down on people being _vague_ about opportunities so we don't catch them in the act.

**Comment by samlachance** [comment]
Sadly discord could solve this easily since they have that role system. They could just be banished to the shadowrealm of opportunities and call it done but where it starts to affect the community is when they are basically bald faced soliciting in a channel that is meant for discussion and the like
